src/test/java/nurseybook/model/NurseyBookTest.java

```diff
@@ -127,6 +128,13 @@ public void getTaskList_modifyList_throwsUnsupportedOperationException() {
             return tasks;
         }
 
+        @Override
+        public ObservableList<Task> getRealTaskList() {
+            //TODO Is this wrong?
```

Comment:
Not sure if this comment is supposed to be here but it seems that tasks is an empty list. I feel that since it is a stub, perhaps we can just return tasks?

Reply:
I agree it makes more sense to just return tasks. I'll change it accordingly!